util/annotations/helpers.go

```diff
@@ -29,19 +29,25 @@ func IsPaused(cluster *clusterv1.Cluster, o metav1.Object) bool {
 	if cluster.Spec.Paused {
 		return true
 	}
-	return HasPausedAnnotation(o)
+	return HasPaused(o)
 }
 
 // IsExternallyManaged returns true if the object has the `managed-by` annotation.
 func IsExternallyManaged(o metav1.Object) bool {
 	return hasAnnotation(o, clusterv1.ManagedByAnnotation)
 }
 
+//Deprecated: Please use `HasPaused` instead
```

Comment:
Nit: This is missing space after // and final period.

```
// Deprecated: this is going to be removed in a next release. Please use `HasPaused` instead.
```

Reply:
Got it..